raml/sfdc_api.raml

```diff
@@ -0,0 +1,52 @@
+# RAML 0.0
+title:
+version: v1
+baseUrl: http://example.com/api
+/alert:
+  post:
+    description: Create or update Case in SFDC
```

Comment:
POST is used because we are going to create resource=alert. Alert will be created if it does not exist and updated with comment if it exist. Nagios/other service have no access to SFDC so it can't check if alert exist or not.